DEV-6/DEV-6/CommandsInvoker.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DEV_6
+{
+    /// <summary>
+    /// The commands invoker.
+    /// </summary>
+    internal class CommandsInvoker
+    {
+        /// <summary>
+        /// Invokes the commands, based on user's input.
+        /// </summary>
+        /// <param name="listOfCars">
+        /// The list of cars with their info.
+        /// </param>
+        public void InvokeCommands(List<CarInfoStruct> listOfCars)
+        {
+            while (true)
+            {
+                Console.WriteLine("Input a command:");
+                var command = Console.ReadLine();
+                if (!command.Equals("exit"))
```

Comment:
is it better to add one more 'case' for 'switch' with command 'exit'. this way the nesting will be less